Tasks/FileTransformV1/task.json

```diff
@@ -16,9 +16,9 @@
         "DeploymentGroup"
     ],
     "version": {
-        "Major": 1,
-        "Minor": 153,
-        "Patch": 2
+        "Major": 2,
```

Comment:
we need to create a new folder with name `FileTransformV2`.
Also, I would be great to go through the enhancements under `issues` for File Transform and accomodate them in the new task :)

Reply:
Created a new folder. Also, there are no enhancements under issues for File Transform Task right now.